backend/LeetWars.Builder/LeetWars.Builder.csproj

```diff
@@ -16,19 +16,17 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Remove="RunnerDefaults\CSharp\CSharpRunner\Program.txt" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <None Include="RunnerDefaults\CSharp\CSharpRunner\CSharpRunner.csproj" />
+    <None Include="RunnerDefaults\CSharp\CSharpTestRunner\SolutionTester.Tests\SolutionTester.Tests.csproj" />
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="Docker.DotNet" Version="3.125.15" />
     <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.19.4" />
+    <PackageReference Include="SharpCompress" Version="0.33.0" />
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\LeetWars.Core\LeetWars.Core.Common\LeetWars.Core.Common.csproj" />
```

Comment:
Need to add to Dockerfile

Reply:
All references should be added to dockerfile, so they can work separatly

Reply:
Now there is no connection from Builder to Common